Microsoft.Dynamics365.UIAutomation.Api/Pages/Navigation.cs

```diff
@@ -441,10 +454,18 @@ public BrowserCommandResult<bool> OpenPrivacyStatement(int thinkTime = Constants
 
                 var subItems = subNavElement.FindElements(By.ClassName(Elements.CssClass[Reference.Navigation.SubActionElementClass]));
 
-                foreach (var subItem in subItems)
+                foreach (var subItem in subItems )
```

Comment:
I have 52 Items in the Navigation Panel and 25 are visible in the first Pane, rest all are hidden in the Sub Area. 
The var subItems is returning 52 elements but only 25 entities which are visible in the first have "text" 
For rest of them the text value is returning as empty. I am trying this in chrome. But still facing the issue.